src/Admin/AbstractAdmin.php

```diff
@@ -1392,6 +1401,12 @@ public function setTemplate($name, $template)
      */
     public function getTemplates()
     {
+        @trigger_error(sprintf(
+            'Method "%s" is deprecated since sonata-project/admin-bundle 3.34 and will be removed in 4.0. '
+            .'Use TemplateRegistry services instead.',
```

Comment:
```suggestion
            'Method "%s()" is deprecated since sonata-project/admin-bundle 3.34 and will be removed in 4.0.'
            .' Use TemplateRegistry services instead.',
```